drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -222,6 +242,7 @@
     display: none;
 }
 .xblock--drag-and-drop--editor .items-form .row.advanced-link {
+    cursor: pointer;
```

Comment:
This is because the `<a>` element has no `href=""` attribute, right? Is it better to do this or use `href="#"` + `preventDefault()` ?